src/styles/components/forms/_element.scss

```diff
@@ -23,58 +18,35 @@
     border: 0;
     margin: 0;
     padding: $input-padding;
-    color: $mc-color-light;
+    color: var(--mc-theme-form-elem-color);
     width: 100%;
     font-size: 16px;
     line-height: 1.5;
     letter-spacing: 0.625px;
     text-overflow: ellipsis;
     resize: none;
 
-    .mc-invert & {
-      color: $mc-color-dark;
-    }
-
     &::placeholder {
-      color: rgba($mc-color-light, 0.5);
-
-      .mc-invert & {
-        color: rgba($mc-color-dark, 0.3);
-      }
+      color: var(--mc-theme-form-elem-placehold-color);
     }
   }
 
   &--focus {
-    color: $mc-color-gray-600;
+    color: var(--mc-theme-form-focus-color);
```

Comment:
not sure this one is defined

Reply:
good catch - this was unnecessary.  reverted